frontend/src/index.css

```diff
@@ -55,3 +54,29 @@
     }
   }
 }
+
+/** Loader **/
+
+.cls-1 {
+  fill: none;
+  stroke: #003c57;
+  stroke-width: 10px;
+  stroke-dasharray: 1400;
+  stroke-dashoffset: 1400;
+  animation: draw 2.5s linear infinite;
+}
+svg {
```

Comment:
Please remove this, this will apply for all the `svg` elements.
Instead you can pass the width to the loader component directly with tailwind.